### DISCUSSION TITLE
[Question][Hw3]

### DISCUSSION THREAD

**Discussion by 1a2babystar** [prosyslab-classroom/cs348-information-security#513] [Q&A]
when I try to compile the root directory and I command make but it said 

make: dune: Command not found

What is the problem?

thank you

**Comment by RiceBiscuits** [comment]
It seems that dune is not installed on your system.

**Comment by RiceBiscuits** [comment]
Please check the link below and install Dune in your system.
(https://docs.google.com/presentation/d/1y5Pk-Ks3883ut5WqYRnj8YggxBiAK3ZVzf-hJN6Z8XY/edit?usp=sharing)

**Comment by Standchen** [comment]
You need to install dune. Try this:

opam init --compiler=4.11.0 --disable-sandboxing
opam switch create cs448-4.11.0 4.11.0
eval $(opam env)
opam install dune llvm.10.0.0 ounit

**Comment by 1a2babystar** [comment]
Ok
That's solve the problem.
Thank you very much!!
